webui/tests/tests/StepImplementation.ts

```diff
@@ -293,4 +294,10 @@ export default class StepImplementation {
     const numAgents = (await stats[0].text()).replace('Number of Agents', '');
     assert.strictEqual(parseInt(numAgents), parseInt(count));
   }
+
+  /* Logs */
+  @Step('Should have some log entries')
+  public async checkSomeLogLines() {
+    await assert.ok((await $('[class*=LogViewer_line]').elements()).length > 0);
```

Comment:
**chore:** You can drop the first await in front of `assert`.

Reply:
**suggestion:** One thought is to look for a particular log text (like `master configuration:` as the very first log line).
```
assert.ok(await text(/^master configuration:/i).exists());
```

Reply:
Just let me know, I'm not sure if that always appear or not, I thought if we have > 1 lines, the page is working, so that should be good?

Reply:
I think it's possible that we might move away from the log viewer (so the class match will break in that case) and less likely for the `master configuration:` line to go away in the future.

Reply:
That is/will not work because `master configuration:` is the 1st line in the log, but when Taiko tests get to the point of testing the master logs page, there are already 100-200 lines of logs, which means that the 1st line text is not in the DOM because log viewer will not render lines out of the viewport.

I think we should stick with: `assert.ok((await $('[class*=LogViewer_line]').elements()).length > 0);`. Once that classname get changed the test will fail and will get fixed?

Reply:
ah you're absolutely right. nm then!